rs/bitcoin/kyt/Cargo.toml

```diff
@@ -1,13 +1,21 @@
 [package]
-name = "btc-kyt-canister"
+name = "ic-btc-kyt"
 version.workspace = true
 authors.workspace = true
 edition.workspace = true
 description.workspace = true
 documentation.workspace = true
 
+[[bin]]
+name = "ic-btc-kyt-canister"
+path = "src/main.rs"
+
 [dependencies]
-bitcoin = { workspace = true }
+bitcoin = { version = "0.32.2", default-features = false }
 candid = { workspace = true }
 futures = { workspace = true }
 ic-cdk = { workspace = true }
+
+[dev-dependencies]
```

Comment:
nit: Shouldn't it also contain the `pocket-ic` and `pocket-ic-server` dependencies like in Bazel?

Reply:
The pocket-ic-server isn't required for compilation. But for bazel, it is required due to runtime dependency on the pocket-ic-server. 

For cargo, specifying it as a dev-dependency won't help with runtime anyway. So I think it is safe to omit.

Reply:
Thanks for the explanation!